<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hellllllloo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../img-removebg-preview.png</normaloff>../../../../img-removebg-preview.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(12,12,12);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>701</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(12 ,12, 12);
font: 75 15pt &quot;Quicksand&quot;;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>310</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;
color:white</string>
     </property>
     <property name="text">
      <string>Email ID</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>320</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom:3px solid qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.0379147 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 80, 0, 255), stop:0.312796 rgba(181, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.50237 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(0, 15, 255, 255), stop:1 rgba(252, 0, 255, 255));
color:white</string>
     </property>
     <property name="text">
      <string>Email ID</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>161</width>
       <height>161</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images/image8.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;
text-decoration: underline;
font: 75 14pt &quot;Quicksand&quot;;
color:white;</string>
     </property>
     <property name="text">
      <string>Log Out</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 50pt &quot;Quicksand&quot;;
border-bottom:3px solid qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.0379147 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 80, 0, 255), stop:0.312796 rgba(181, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.50237 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(0, 15, 255, 255), stop:1 rgba(252, 0, 255, 255));
color:white;
</string>
     </property>
     <property name="text">
      <string>Syncer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>190</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;
color:white</string>
     </property>
     <property name="text">
      <string>Device Name</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>200</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom:3px solid qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.0379147 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 80, 0, 255), stop:0.312796 rgba(181, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.50237 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(0, 15, 255, 255), stop:1 rgba(252, 0, 255, 255));
color:white</string>
     </property>
     <property name="text">
      <string>Device1 Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>260</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom:3px solid qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.0379147 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 80, 0, 255), stop:0.312796 rgba(181, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.50237 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(0, 15, 255, 255), stop:1 rgba(252, 0, 255, 255));
color:white</string>
     </property>
     <property name="text">
      <string>Device2 Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>250</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;
color:white</string>
     </property>
     <property name="text">
      <string>Device Name</string>
     </property>
    </widget>
    <widget class="QFrame" name="circularProgressBar_Main">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>390</y>
       <width>240</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="cpucircularprogress_b">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>220</width>
        <height>220</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame
{
border-radius:110px;
background-color:rgba(77,77,127,120);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="cpucircularprogress_a">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>220</width>
         <height>220</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame
{
border-radius:110%;
background-color: qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0.749 rgba(255, 255, 255, 0), stop:0.750 rgba(85, 170, 255, 255));
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="cpucircularprogress_c">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>24</y>
        <width>191</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
border-radius:95px;
background-color:rgb(58, 58, 102);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="cpu_used">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>73</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(115, 185, 255); padding: 0px; background-color: none;
font: 75 26pt &quot;Quicksand&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="cpu_total">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>143</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(148, 148, 216); background-color: none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="cpu">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF; background-color: none;</string>
       </property>
       <property name="text">
        <string>DEVICE1 USAGE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>640</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:20px;
border:3px solid white;
font: 75 20pt &quot;Quicksand&quot;;
color:white;
</string>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QFrame" name="circularProgressBar_Main_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>390</y>
       <width>240</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="cloudcircularprogress_a">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>220</width>
        <height>220</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
border-radius:110%;
background-color:qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0.649 rgba(255, 255, 255, 0), stop:0.650 rgba(85, 255, 127, 255));
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="cloudcircularprogress_b">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>220</width>
        <height>220</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	border-radius: 110px;	
	background-color: rgba(85, 85, 127, 120);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="cloudcircularprogress_c">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>24</y>
        <width>191</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	border-radius: 95px;	
	background-color: rgb(58, 58, 102);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="cloud_used">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>73</y>
         <width>151</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(115, 255, 171); padding: 0px; background-color: none;
font: 75 26pt &quot;Quicksand&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="cloud">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF; background-color: none;</string>
       </property>
       <property name="text">
        <string>CLOUD USAGE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="cloud_total">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>143</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(148, 148, 216); background-color: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <zorder>cloudcircularprogress_b</zorder>
     <zorder>cloudcircularprogress_a</zorder>
     <zorder>cloudcircularprogress_c</zorder>
    </widget>
    <widget class="QFrame" name="circularProgressBar_Main_3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>390</y>
       <width>240</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="cloudcircularprogress1_a">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>220</width>
        <height>220</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
border-radius:110%;
background-color:qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0.649 rgba(255, 255, 255, 0), stop:0.65 rgba(255, 0, 127, 255));
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="cloudcircularprogress1_b">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>220</width>
        <height>220</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	border-radius: 110px;	
	background-color: rgba(85, 85, 127, 120);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="cloudcircularprogress1_c">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>24</y>
        <width>191</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	border-radius: 95px;	
	background-color: rgb(58, 58, 102);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="cloud2_used">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>73</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 44, 174); padding: 0px; background-color: none;
font: 75 26pt &quot;Quicksand&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="cloud2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF; background-color: none;</string>
       </property>
       <property name="text">
        <string>DEVICE2 USAGE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="cloud2_total">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>143</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(148, 148, 216); background-color: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <zorder>cloudcircularprogress1_b</zorder>
     <zorder>cloudcircularprogress1_a</zorder>
     <zorder>cloudcircularprogress1_c</zorder>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>690</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 12,12,12</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
